frontend/src/pages/pipelines/global/runs/ActiveRuns.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import {
+  Bullseye,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  Spinner,
+  EmptyStateHeader,
+  EmptyStateFooter,
+  EmptyStateActions,
+  Button,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon, PlusCircleIcon } from '@patternfly/react-icons';
+
+import PipelineRunTable from '~/concepts/pipelines/content/tables/pipelineRun/PipelineRunTable';
+import { usePipelineActiveRunsTable } from '~/concepts/pipelines/content/tables/pipelineRun/usePipelineRunTable';
+import { PipelineRunSearchParam } from '~/concepts/pipelines/content/types';
+import { PipelineRunType } from './types';
+
+export const ActiveRuns: React.FC = () => {
+  const navigate = useNavigate();
+  const { namespace } = useParams();
+  const [[{ items: runs, totalSize }, loaded, error], { initialLoaded, ...tableProps }] =
+    usePipelineActiveRunsTable();
+
+  if (error) {
+    return (
+      <Bullseye>
+        <EmptyState>
+          <EmptyStateHeader
+            titleText="There was an issue loading active runs"
+            icon={<EmptyStateIcon icon={ExclamationCircleIcon} />}
+            headingLevel="h2"
+          />
+          <EmptyStateBody>{error.message}</EmptyStateBody>
+        </EmptyState>
+      </Bullseye>
+    );
+  }
+
+  if (!loaded && !initialLoaded) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
+
+  if (loaded && totalSize === 0 && !tableProps.filter) {
+    return (
+      <EmptyState data-testid="active-runs-empty-state">
+        <EmptyStateHeader
+          titleText="No active runs"
+          icon={<EmptyStateIcon icon={PlusCircleIcon} />}
+          headingLevel="h2"
+        />
+
+        <EmptyStateBody>
+          To get started, create a run. Alternatively, click the <b>Schedule</b> tab to create,
```

Comment:
```suggestion
          To get started, create a run. Alternatively, click the <b>Schedules</b> tab to create,
```
Anyway for us to use a constant here? We are referring to the literal tab naming -- and it may be annoying to compare the two. Maybe this is just a test? seems like a crutch for a variable though -- I'll leave it up to you, but either way the typo exists

Reply:
FWIW, this causes a rendering issue under the hood -- https://issues.redhat.com/browse/RHOAIENG-3426, unrelated to your implementation here -- but part of the tab CSS.